user-guide/Advanced_Modules/DOM/DOM_status_system.md

```diff
@@ -144,6 +144,7 @@ A `DomStatusFieldDescriptorLink` has the following properties:
 >
 > - If no `FieldDescriptorLink` is present for an existing `FieldDescriptor`, no values are allowed to be present for this `FieldDescriptor` when the `DomInstance` has that specific status.
 > - To prevent performance issues, when a `DomBehaviorDefinition` is saved, there is no check whether the `SectionDefinitions` and `FieldDescriptors` exist.
+> - When the 'ReadOnly' or 'ClientReadOnly' option is enabled, the field will still be assignable in a Low-Code App's create form (but not in the edit form). This behavior is consistent with the non-status system, where it would otherwise be impossible to assign a value. If a field should be hidden during the create form, you must set it as hidden in the initial status and make it visible in subsequent states.
```

Comment:
The "edit form" and "create form" aren't entirely clear to me. Is this what you mean?
```suggestion
> - When the 'ReadOnly' or 'ClientReadOnly' option is enabled, it will be possible to assign the field in a low-code app form when a new instance is created but not when an instance is edited. This behavior is consistent with the non-status system, where it would otherwise be impossible to assign a value. If a field should be hidden when an instance is created, you must set it as hidden in the initial state and make it visible in subsequent states.
```

Reply:
Yes, that is indeed clearer. I referred to the fact that the form component for a DOM instance in a LCA can be in a 'Create' mode when it is fed by a DOM definition, or an 'Edit' mode when it is fed by a DOM instance.